lms/djangoapps/courseware/tests/test_views.py

```diff
@@ -521,7 +519,15 @@ def test_user_groups(self):
         mock_user.is_authenticated.return_value = False
         self.assertEqual(views.user_groups(mock_user), [])
 
+    # TODO: TNL-6546: Remove decorator for unified_course_view
+    @override_flag('unified_course_view', active=True)
     def test_get_redirect_url(self):
+        # test the course location
+        self.assertIn(
+            'activate_block_id',
+            get_redirect_url(self.course_key, self.course.location),
```

Comment:
Reviewer: In order to get this test to properly fail when the bug existed, I needed to temporarily add a third argument of True, because get_redirect_url had an additional argument named unified_course_view.  As written, this wouldn't have failed at the time, but just adding the decorator for fun I guess. :)

Here is the error I saw when running on a commit before the revert:
```
AssertionError: 'activate_block_id' not found in u'/courses/org.0/course_0/Testing_course/course/'
```